src/utilities/telegram_utils.rs

```diff
@@ -22,9 +21,48 @@ impl MainUsername for User {
     }
 }
 
-pub struct MessageImage {
-    pub file: File,
-    pub mime_type: Cow<'static, str>,
+pub enum MessageAttachment {
+    Photo(Photo),
+    Video(Video),
+    Animation(Animation),
+    Audio(Audio),
+    Voice(VoiceNote),
+    Document(Document),
+}
```

Comment:
sort these as tdlib does, and add additional content types

```suggestion
pub enum MessageAttachment {
    Animation(Animation),
    Audio(Audio),
    Document(Document),
    Photo(Photo),
    Sticker(Sticker),
    Video(Video),
    VideoNote(VideoNote),
    VoiceNote(VoiceNote),
    Story(Story),
    ChatChangePhoto(ChatChangePhoto),
    SuggestProfilePhoto(SuggestProfilePhoto),
}
```

Reply:
last two are enums, not types

Reply:
are you importing the correct structs https://docs.rs/tdlib/latest/tdlib/types/struct.MessageChatChangePhoto.html, https://docs.rs/tdlib/latest/tdlib/types/struct.MessageSuggestProfilePhoto.html

Reply:
okay now I see it but after further testing the last two doesn't have a `File`